src/store/chart/chartReducer.js

```diff
@@ -0,0 +1,25 @@
+const initialState = {
+    chartMarket: [],
+    chartLabel: [],
+    chartMarketPrices: [],
+    chartLabelPrices: []
+}
+
+export const GET_CHART_INFO = "GET_CHART_INFO";
+
+function chartReducer(state=initialState, action){
+     switch(action.type){
+        case GET_CHART_INFO:
+            return{
+                ...state,
+                chartMarket: action.payload.total_volumes.map((el) => el[1]),
+                chartLabel: action.payload.total_volumes.map((el) => new Date(el[0]).getDate()),
+                chartMarketPrices: action.payload.prices.map((el) => el[1]),
+                chartLabelPrices: action.payload.prices.map((el) => new Date(el[0]).getDate())
```

Comment:
to be fair, i think maybe you should process the data in your action and send it to your reducer. The reason why I'm saying this is because your reducers can become pretty clogged. But it's your call